extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -50,6 +61,8 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
     super(element);
     /** @private @const {?../../amp-story/1.0/amp-story-store-service.AmpStoryStoreService} */
     this.storeService_ = null;
+    /** @param {!Element} element */
```

Comment:
nit: Please add spaces between properties in constructor to follow style of other components.